api/src/test/java/org/pmiops/workbench/cdr/dao/CBCriteriaAttributeDaoTest.java

```diff
@@ -8,20 +8,13 @@
 import org.junit.runner.RunWith;
 import org.pmiops.workbench.cdr.model.DbCriteriaAttribute;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration;
-import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-import org.springframework.context.annotation.Import;
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.transaction.annotation.Transactional;
 
 @RunWith(SpringRunner.class)
 @DataJpaTest
-@Import(LiquibaseAutoConfiguration.class)
```

Comment:
Can you please add to your description why these are no longer needed (or were never needed)? Should these come out from all the tests in the backend?

aside: I've noticed that a couple of my tests are happier with this import (or at least IntelliJ warnings go away with it present), even if they don't interact with any DAOs at all. I'm wondering if there's some side effect of this configuration that I should be pulling in.

Reply:
Updated the description. I don't think they are hurting anything by having them in the dao test but they can be removed. I don't see any problem in Intellij.